app/components/concept-admin/questions-page/question-form.ts

```diff
@@ -39,6 +43,30 @@ export default class QuestionFormComponent extends Component<Signature> {
   async handleOptionDeleted(optionIndex: number) {
     this.args.question.options = this.args.question.options.filter((_, currentOptionIndex) => currentOptionIndex !== optionIndex);
   }
+
+  @action
+  moveOptionDown(optionIndex: number) {
+    const options = this.options.slice();
+
+    if (optionIndex < options.length - 1) {
```

Comment:
This is a good candidate for an early return @libmartinito - https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement